packages/foundation-models/src/azure-openai/client/inference/schema/azure-cosmos-db-chat-extension-configuration.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2024 SAP SE or an SAP affiliate company. All rights reserved.
+ *
+ * This is a generated file powered by the SAP Cloud SDK for JavaScript.
+ */
+import type { AzureOpenAiAzureChatExtensionConfiguration } from './azure-chat-extension-configuration.js';
+import type { AzureOpenAiAzureCosmosDBChatExtensionParameters } from './azure-cosmos-db-chat-extension-parameters.js';
+/**
+ * A specific representation of configurable options for Azure Cosmos DB when using it as an Azure OpenAI chat
+ * extension.
+ */
+export type AzureOpenAiAzureCosmosDBChatExtensionConfiguration =
```

Comment:
Only the relevant chat completion types are currently exported from the package.

I am honestly not sure about deleting them manually, the next time we update the spec we have the same problem. We dont delete unused types in Orchestration as well. 
@marikaner what do you think?

Reply:
I think the idea would be to add a `&& rm ...` to the script, or exclude it from the export. For orchestration we probably need to do the same, as we otherwise publish unreleased and non-functional API.

Reply:
Right now we exclude the export of unused types from both the packages (see index.ts). So even though we publish everything, users will not be able to access these types using @sap-ai-sdk/foundation-models.

I did this manually for now, maybe having a script would be a better (which either deletes the files or adds exports to index.ts).